test/hybig/HyBIG_Regression.ipynb

```diff
@@ -0,0 +1,467 @@
+    "                      'ASTGTMV003_N00E022_dem.pgw',\n",
+    "                      'ASTGTMV003_N00E022_dem.png.aux.xml']\n",
+    "\n",
+    "\n",
+    "    with TemporaryDirectory() as temp_dir:\n",
+    "        downloaded_grid_outputs = [\n",
+    "            file_future.result()\n",
+    "            for file_future\n",
+    "            in harmony_client.download_all(aster_job_id, overwrite=True, directory=temp_dir)\n",
+    "        ]\n",
+    "        \n",
+    "        \n",
+    "        reference_dir = Path('./regression_data')\n",
+    "\n",
+    "        test_files = list(map(lambda fn: Path(temp_dir) / fn, expected_files))\n",
+    "        reference_files = list(map(lambda fn: reference_dir / fn, expected_files))\n",
+    "\n",
+    "        for file_name in test_files:\n",
+    "            assert file_name.exists() == True, f'File does not exist {file_name.resolve()}'\n",
+    "        print_success('all test files generated')\n",
+    "\n",
+    "        expected_output_metadata = {'driver': 'PNG',\n",
+    "                                    'dtype': 'uint8',\n",
+    "                                    'nodata': None,\n",
+    "                                    'width': 3601,\n",
+    "                                    'height': 3601,\n",
+    "                                    'count': 4,\n",
+    "                                    'crs': CRS.from_epsg(4326),\n",
+    "                                    'transform': Affine(0.000277777777777778, 0.0, 21.9998611111111,\n",
+    "                                                        0.0, -0.000277777777777778, 1.00013888888889)}\n",
+    "\n",
+    "        test_dataset = rasterio.open(test_files[0])\n",
+    "        assert test_dataset.meta == expected_output_metadata, 'output PNG has incorrect metadata'\n",
+    "        print_success('Generated image has correct metadata.')\n",
+    "\n",
+    "        reference_dataset = rasterio.open(reference_files[0])\n",
+    "\n",
+    "        ref_image = reference_dataset.read()\n",
+    "        test_image = test_dataset.read()\n",
+    "        assert_array_almost_equal(ref_image, test_image)\n",
+    "        print_success('Generated image contains correct data.')\n",
+    "\n",
+    "\n",
+    "    print_success('Conversion of ASTER Geotiff to PNG Success')\n",
+    "else:\n",
+    "    print('Skipping test: HyBIG regression tests not configured for this environment.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1edc6d53",
+   "metadata": {},
+   "source": [
+    "### Test a request for JPG output from a 1-band GeoTIFF with no color information\n",
+    "\n",
+    "Forms a request for ASTER data as 'image/png', ensures all files are created that the image has the correct metadata and that that the data in the png file matches the reference data in the test.\n",
+    "\n",
+    "*This test has to fix the downloaded aux.xml file becuase it is downloaded as gzipped but named as if it is not*"
```

Comment:
Nit: The workaround has been removed, so this text can probably go, too.

Reply:
f9cdca4